src/format_config.mli

```diff
@@ -0,0 +1,22 @@
+open! Stdune
```

Comment:
It's nice to have a comment at the beginning of each toplevel module, just so that an external reader can quickly identify what the module is about:

```suggestion
(** Represent the [format_config] field in [dune-project] files *)

open! Stdune
```